src/app/app.component.scss

```diff
@@ -0,0 +1,7 @@
+h1,a {
+    margin: 10px 10px 10px 10px;
+}
+
+.btn-no-waves .btn.waves-effect {
+    position: static;
```

Comment:
Why did you add those styles? That's weird

Reply:
It was added to one of parts of mdb as a style, but in fact it doesn't change anything :D I've deleted